src/main/resources/org/gradle/profiler/report/report-template.html

```diff
@@ -0,0 +1,324 @@
+                    <input type="radio" name="sorting" value="historical" autocomplete="off" checked> Historical
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="sorting" value="sorted" autocomplete="off"> Sorted
+                </label>
+            </div>
+        </div>
+        <div class="form-group col-md-1 col-form-label">
+            <label for="sample">Samples:</label>
+        </div>
+        <div class="form-group col-md-8">
+            <select multiple class="form-control" id="sample">
+            </select>
+        </div>
+    </div>
+    <div class="row mt-3">
+        <div class='col ml-auto mr-auto'>
+          <div id="data-table"></div>
+        </div>
+    </div>
+</div>
+<script>
+Array.prototype.unique = function() { return [...new Set(this)]; };
+
+function iterations(scenario, phase) {
+    return scenario.iterations.filter(iteration => iteration.phase === phase);
+}
+function iterationTitles() {
+    return benchmarkResult.scenarios
+        .map(scenario => scenario.iterations
+            .map(iteration => iteration.title)
+        )
+        .flat()
+        .unique();
+}
+function sort(array) {
+    return array.slice().sort((a, b) => a - b);
+}
+
+function quantile(data, q) {
+    const sorted = sort(data);
+    const pos = (sorted.length - 1) * q;
+    const base = Math.floor(pos);
+    const rest = pos - base;
+    if (sorted[base + 1] !== undefined) {
+        return sorted[base] + rest * (sorted[base + 1] - sorted[base]);
+    } else {
+        return sorted[base];
+    }
+};
+
+function mean(data) {
+    return data.reduce((sum, val) => sum += val) / data.length;
+}
+
+const OPERATIONS = {
+    "mean": (data) => mean(data),
+    "min": (data) => Math.min(...data),
+    "max": (data) => Math.max(...data),
+    "p25": (data) => quantile(data, .25),
+    "median": (data) => quantile(data, .50),
+    "p75": (data) => quantile(data, .75),
+    "stddev": (data) => {
+        let dataMean = mean(data);
+        return Math.sqrt(mean(data.map(x => Math.pow(x - dataMean, 2))));
+    }
+};
+
+Handlebars.registerHelper("sampleCount", function(scenario) {
+    return scenario.samples.length;
+});
+
+Handlebars.registerHelper("percent", function(percent) {
+    return (percent * 100).toFixed(2) + "%";
+});
+
+Handlebars.registerHelper("number", function(value) {
+    return value.toFixed(2);
+});
+
+Handlebars.registerHelper("calc", function(operation, sample, scenario) {
+    let data = iterations(scenario, Phase.MEASURE).map(iteration => iteration.values[sample.name]);
+    return OPERATIONS[operation](data).toFixed(2);
+});
+
+let source = document.getElementById("data-table-template");
+let target = document.getElementById("data-table");
+let template = Handlebars.compile(source.innerHTML);
+let maxIterations = Math.max(...(benchmarkResult.scenarios.map(scenario => scenario.iterations.length)));
```

Comment:
`maxIterations` doesn't seem to be used.

Reply:
Removed.